web-app/src/components/CreateClientForm/index.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import container from './container';
+import employeeStatus from '../../utilities/employeeStatus';
+import { Form, Input } from '../common';
+import { FormContainer } from './styled';
+
+export const CreateClientForm = props => {
+  const {
+    clientId,
+    submitFormCreate,
+    submitFormUpdate,
+    formStatus,
+    formData,
+    formIsValid,
+  } = props;
+
+  let ctas = [];
+  if (clientId > 0) {
+    ctas = [
+      {
+        label: 'Update Client',
+        event: submitFormUpdate,
+        disabled: false,
+      },
+    ];
+  } else {
+    ctas = [
+      {
+        label: 'Create Client',
+        event: submitFormCreate,
+        disabled: !formIsValid,
+      },
+    ];
+  }
+
+  return (
+    <FormContainer>
+      <Form formData={formData} formStatus={formStatus} actions={ctas}>
+        <Input
+          type="input"
+          htmlAttrs={{
+            type: 'input',
+            name: 'clientName',
+            placeholder: 'Enter a Client Name',
+          }}
+          value={formData.clientName}
+          focus
+          label="Client Name:"
+          rules={{
+            required: true,
+          }}
+        />
+        <Input
+          type="select"
+          htmlAttrs={{
+            name: 'clientStatusId',
+            options: [
+              { value: employeeStatus.ACTIVE, displayValue: 'Active' },
+              { value: employeeStatus.INACTIVE, displayValue: 'Inactive' },
+            ],
+          }}
+          value={formData.clientStatusId}
+          label="Client Status Id:"
+        />
+        <Input
+          type="textarea"
+          htmlAttrs={{
+            type: 'input',
+            name: 'clientStatusDescription',
+            placeholder: 'Enter a client Status Description',
+          }}
+          value={formData.clientStatusDescription}
+          label="Client Status Description:"
+          rules={{
+            required: true,
+          }}
+        />
+        <Input
+          type="input"
+          htmlAttrs={{
+            type: 'email',
+            name: 'contactEmail',
+            placeholder: 'Enter an Contact Email',
+          }}
+          value={formData.contactEmail}
+          label="Contact Email:"
+          rules={{
+            required: true,
+            isEmail: true,
+          }}
+        />
+        <Input
+          type="input"
+          htmlAttrs={{
+            type: 'input',
+            name: 'contactName',
+            placeholder: 'Enter an Contact Name',
+          }}
+          value={formData.contactName}
+          label="Contact Name:"
+          rules={{
+            required: true,
+          }}
+        />
+        <Input
+          type="select"
```

Comment:
Not sure a select box is the most optimal thing in terms of UX for this type of input

Reply:
Updated this to a input field with a type of number